modules/release-notes/ref-release-notes-new-features.adoc

```diff
@@ -4,5 +4,65 @@
 
 This section highlights new features in {product} {product-version}.
 
-None.
+[id="feature-rhidp-3597"]
+== OpenTelemetry metrics support added to the Keycloak backend plugin
+
+With this update, the Keycloak backend plugin supports OpenTelemetry metrics, which monitors fetch operations and diagnoses potential issues.
+
+The available counters include the following:
+
+`backend_keycloak_fetch_task_failure_count_total`: Counts fetch task failures where no data was returned due to an error.​
+
+`backend_keycloak_fetch_data_batch_failure_count_total`: Counts partial data batch failures. Even if some batches fail, the plugin continues fetching others.​
+
+These counters include the `taskInstanceId` label, which uniquely identifies each scheduled fetch task, and allows you to trace failures back to individual task executions.
+
+Example configuration:
+
+```text
+backend_keycloak_fetch_data_batch_failure_count_total{taskInstanceId=&#34;df040f82-2e80-44bd-83b0-06a984ca05ba&#34;} 1
+```
+
+You can export metrics using any OpenTelemetry-compatible backend, such as **Prometheus**.
+
+
+[id="enhancement-rhidp-5987"]
+== Updated RBAC and Bulk Import plugins

```

Comment:
I think we should rename this to "Updated Auditor Service".  We can mention the RHDH plugins that are impacted but the core components (Catalog and Scaffolder) are also affected too.  cc @schultzp2020